### ISSUE TITLE
Alt text missing from buttons on the task board

### ISSUE THREAD

**Issue by marciogurka** [bryntum/support#9360]
[Forum post](https://forum.bryntum.com/viewtopic.php?f=51&t=29341&p=148123#p148123)

"Several of the buttons on the task board do not have alternative text. 

For example,

The lane expand button ![Screenshot 2024-06-10 at 17.15.45.png](https://forum.bryntum.com/download/file.php?id=28961) and the menu button ![Screenshot 2024-06-10 at 17.19.08.png](https://forum.bryntum.com/download/file.php?id=28962).

Could alternative text/ an accessibility name please be added to these buttons to improve the performance of screen readers."

**Comment by ExtAnimal** [comment]
In all buttons, setting the `aria-label` property to the same value that `data-btip` has will give screen readers the information they need.
